src/ArraysAndLoops.java

```diff
@@ -0,0 +1,102 @@
+import java.util.Arrays;
+
+public class ArraysAndLoops {
+    public static void main(String[] args) {
+        int [] a = {1, 1, 1, 6564, 2, 1};
+        balanceSum(a);
+    }
+
+    public static void changingArrays(int [] a){
+        for(int i = 0; i < a.length; i++){
+            if(a[i] == 0){
+                a[i] = 1;
+            } else a[i] = 0;
```

Comment:
Скобки ставятся или не ставятся. Best practices говорит, что надо всегда ставить

```suggestion
            if (a[i] == 0) {
                a[i] = 1;
            } else { 
                a[i] = 0;
            }
```